data/schema.js

```diff
@@ -0,0 +1,34 @@
+import { buildSchema, GraphQLList } from 'graphql';
```

Comment:
I think that in my app this file was generated from other files - not sure if that is the case here but if so it should not be included in source control.

Reply:
schema.js is our own file, it's not generated. schema.graphql is the generated file and the way we handle that is different. You've generated it through the updateSchema.js script. I, on the other hand, have generated it with the buildSchema method in 'graphql' module.